engine/common/src/test/java/org/enso/common/EnsoSourcesTest.java

```diff
@@ -0,0 +1,78 @@
+package org.enso.common;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
+import org.junit.Test;
+
+public class EnsoSourcesTest {
+  @Test
+  public void verifyEngineCommonSourcesFile() throws Exception {
+    var url = EnsoSourcesTest.class.getProtectionDomain().getCodeSource().getLocation();
+    var where = new File(url.toURI());
```

Comment:
Massive introduction of `module-info.java` done in #10823 confuses our [VSCode & IGV support](https://github.com/enso-org/enso/blob/develop/tools/enso4igv/README.md) and breaks debugging. Writing a test to ensure the `input` property is properly computed and similar regressions are avoided next time.

Reply:
In addition to the test, here is a visual verification. Projects are found and there are no errors among them:

![no errors in Enso projects](https://github.com/user-attachments/assets/d9167364-8e4d-4e7e-b05e-d2f586b1ca29)